org-formation/070-guard-duty/_tasks.yaml

```diff
@@ -23,24 +23,24 @@ GuardDuty:
       Account: !Ref LogCentralAccount
     MemberBinding:
       Account:
-        - !Ref BmgfkiAccount
-        - !Ref BridgeProdAccount
-        - !Ref ScicompAccount
-        - !Ref ScipoolProdAccount
-        - !Ref SynapseProdAccount
-        - !Ref WorkflowsNextflowDevAccount
-        - !Ref WorkflowsNextflowProdAccount
+        - !Ref BridgeOU
+        - !Ref ItProdOU
+        - !Ref PolicyStagingOU
+        - !Ref ServiceCatalogOU
+        - !Ref ScienceDevOU
+        - !Ref ScienceProdOU
+        - !Ref SynapseOU
       IncludeMasterAccount: false
     AllBinding:
       Account:
         - !Ref accountId
-        - !Ref BmgfkiAccount
-        - !Ref BridgeProdAccount
-        - !Ref ScicompAccount
-        - !Ref ScipoolProdAccount
-        - !Ref SynapseProdAccount
-        - !Ref WorkflowsNextflowDevAccount
-        - !Ref WorkflowsNextflowProdAccount
+        - !Ref BridgeOU
+        - !Ref ItProdOU
+        - !Ref PolicyStagingOU
```

Comment:
Should this account be in the binding? Not sure if you want to treat it differently as a staging env.

Reply:
removed PolicyStagingOU